pages/app/controllers/refinery/pages_controller.rb

```diff
@@ -10,6 +10,9 @@ class PagesController < ::ApplicationController
 
     # This action is usually accessed with the root path, normally '/'
     def home
+      if page.link_url.present? && page.link_url != "/"
```

Comment:
Do you also allow a redirect of the homepage if there is a path in `link_url` ? It's nice!

Reply:
fixed as recommended